x-pack/test/functional/apps/rollup_job/rollup_jobs.js

```diff
@@ -8,39 +8,49 @@
 import datemath from '@kbn/datemath';
 import expect from '@kbn/expect';
 import { mockIndices } from './hybrid_index_helper';
+// import { FtrProviderContext } from '../../ftr_provider_context';
 
 export default function ({ getService, getPageObjects }) {
-  const es = getService('es');
+  const config = getService('config');
   const PageObjects = getPageObjects(['rollup', 'common', 'security']);
   const security = getService('security');
   const esDeleteAllIndices = getService('esDeleteAllIndices');
   const kibanaServer = getService('kibanaServer');
+  const es = getService('es');
+  const isRunningCcs = config.get('esTestCluster.ccs') ? true : false;
+  let remoteEs;
+  if (isRunningCcs) {
+    remoteEs = getService('remoteEs');
+  }
 
   describe('rollup job', function () {
-    //Since rollups can only be created once with the same name (even if you delete it),
-    //we add the Date.now() to avoid name collision.
+    // Since rollups can only be created once with the same name (even if you delete it),
+    // we add the Date.now() to avoid name collision.
     const rollupJobName = 'rollup-to-be-' + Date.now();
     const targetIndexName = 'rollup-to-be';
-    const rollupSourceIndexPattern = 'to-be*';
+    const indexPatternToUse = 'to-be*';
+    const rollupSourceIndexPattern = isRunningCcs
+      ? 'ftr-remote:' + indexPatternToUse
+      : indexPatternToUse;
     const rollupSourceDataPrepend = 'to-be';
 
-    //make sure all dates have the same concept of "now"
+    // make sure all dates have the same concept of "now"
     const now = new Date();
     const pastDates = [
       datemath.parse('now-1d', { forceNow: now }),
       datemath.parse('now-2d', { forceNow: now }),
       datemath.parse('now-3d', { forceNow: now }),
     ];
     before(async () => {
-      await security.testUser.setRoles(['manage_rollups_role']);
+      await security.testUser.setRoles(['manage_rollups_role', 'global_ccr_role', 'superuser']);
```

Comment:
Is this one of the places you opened an issue for?  I don't think we should add the superuser role to these others.  I was thinking it would be like this where we use superuser instead of the roles that were expected to work;
```
// <issue for security roles not working as expected>
// await security.testUser.setRoles(['manage_rollups_role', 'global_ccr_role']);
await security.testUser.setRoles(['superuser']);
```

Reply:
Note sure if it's covered by this issue? https://github.com/elastic/kibana/issues/143720

Reply:
Fixed